### ISSUE TITLE
"24 Hours Remaining" Coding Partner Review

### ISSUE THREAD

**Issue by Joseph-Burke** [defoebrand/HTML-CSS-Capstone-Project#1]
### Suggestions

- Loyal to the design specification and looks very professional, really good job.

- Carousel is really nice. Maybe tweak the animation to ease in or out ever so slightly.

- Perhaps "Built/Created by Brandon Defoe" would be more appropriate, and then attribute the design to the behance designer.

- Footer categories could be made into links (if only for hover-over appearance). This is one of the more noticeable points for improvement.

- Responsiveness for XL screens could be improved. The search bar continues stretching and the distance between the column of products and the search filters section gets very wide. This is beyond the scope of the project's requirements and would only affect a very small minority of users but could be implemented if you wish.

- _Styling and features of product cards could be brought a bit more in line with the design specification._ **I was going to make this point but as long as the styling from the horizontal cards is applied to the upright cards, then ignore this.**

In summary, the whole thing looks genuinely very professional and is very impressive, I have little doubt it will be swiftly approved.

**Comment by defoebrand** [comment]
All points addressed
